src/user-guide/examples/index.rst

```diff
@@ -0,0 +1,9 @@
+Examples
+========
+
+These examples cover some of the main patterns for implementing Cylc workflows.
+
+.. toctree::
+   :glob:
```

Comment:
TODO: this is a nice way of doing it, but the examples come out in an unhelpful order. E.G. it might be nicer to have `hello_world` at the top.

Reply:
Could include a number or letter at the beginning of the directory names in order to order them?

Reply:
Good idea, I'm not sure if it sorts by file path or page heading, will have a play.

Reply:
I didn't think it would work, but it does!

I'll put up a PR on the cylc-flow end for this.